qmxgraph/mime.py

```diff
@@ -37,6 +37,10 @@
             },
             ...
         ]
+    custom_app_drop_events:  # optional, some drop event in the graph that will be returned to the app for processing
```

Comment:
```suggestion
    custom_app_drop_event:  # if given, a `customAppDropEvent` signal will be emitted with this object
```

Also notice I changed the key to `custom_app_drop_event` to better reflect that we will be sending only one event

Reply:
Hmmm we might even call it `custom_app_drop_event_data` actually, because we this is not the actual event, but the data that will be emitted by it.